doc/developers/advanced_installation.rst

```diff
@@ -0,0 +1,427 @@
+
+.. _advanced-installation:
+
+===================================
+Advanced installation instructions
+===================================
+
+There are different ways to get scikit-learn installed:
+
+  * Install the version of scikit-learn provided by your
+    :ref:`operating system or Python distribution <install_by_distribution>`.
+    This is the quickest option for those who have operating systems that
+    distribute scikit-learn.
+
+  * :ref:`Install an official release <install_official_release>`. This
+    is the best approach for users who want a stable version number
+    and aren't concerned about running a slightly older version of
+    scikit-learn.
+
+  * :ref:`Install the latest development version
+    <install_bleeding_edge>`. This is best for users who want the
+    latest-and-greatest features and aren't afraid of running
+    brand-new code.
+
+.. note::
+
+    If you wish to contribute to the project, it's recommended you
+    :ref:`install the latest development version<install_bleeding_edge>`.
```

Comment:
It's not really recommended. I would say that you need it or that the latest version is necessary.